src/server/users.txt

```diff
@@ -0,0 +1,3 @@
+Email: marcin.serwotka@op.pl, Password: eloelo320
+Email: haber.bartosz@gmail.com, Password: habrex
+Email: kowalski@gmail.com, Password: kowalski123
```

Comment:
Can be simplified to 

marcin.serwotka@op.pl, eloelo320
haber.bartosz@gmail.com, habrex
kowalski@gmail.com, kowalski123

less file size, and nobody will read this file anyway. What's important is only
`<username>, <password>`

moreover you cannot store user data in plain text because everybody can read it. For storing user sensitive data we use hashing:

https://www.freecodecamp.org/news/how-to-hash-passwords-with-bcrypt-in-nodejs/

you can improve this by hashing and then storing: 

`<loginHash>, <passwordHash> `

of course you have to also dehash it when reading from file.

Reply:
done! :)